packages/terra-site/src/examples/spacer/SpacerExample.jsx

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Spacer from 'terra-spacer';
+import Image from 'terra-image';
+import placeholderPic150x150 from '../../assets/150x150.jpg';
+import placeholderPic1000x200 from '../../assets/1000x200.png';
+
+const SpacerExample = () => (
+  <div>
+    <Spacer marginBottom="medium">
+      <Image src={placeholderPic150x150} alt={placeholderPic1000x200} />
```

Comment:
Maybe use buttons in one of your examples since that was the component that had the most need for a way to space them.

Reply:
Updated https://github.com/cerner/terra-core/commit/51079e4e1f7a6e3bb384bba084478bb7fcf12335#diff-2f4ced636a7ac23a1b1498c37d644ae1R116